plugins/example/metadata.yaml

```
@@ -0,0 +1,3 @@
+Name: Example
```

```suggestion
name: Example
```
Please use lowercase for the yaml's key